src/middleware/index.ts

```diff
@@ -9,9 +9,9 @@ import unauthorizedMiddleware from '@/middleware/unauthorized';
  * Applies middleware to express app using old `configure` pattern
  */
 export default function middleware() {
-  this.use(jwtMiddleware());
-  this.use(unauthorizedMiddleware());
+  // this.use(jwtMiddleware());
```

Comment:
how come we're commenting these out? I still don't know enough about the core to really understand what's going on haha

Reply:
So the request can be tested. With the authentication middleware, it's hard to test these routes for now.